client/src/Components/Reviews/Reviews.jsx

```diff
@@ -12,15 +15,33 @@ class Reviews extends React.Component {
   }
 
   render() {
-    const reviews = this.props.reviews.map(review => (
-      <div key={review._id} id="reviews" className="container">
-        {review.name}
-        <img src={review.profilePicture} className="rounded-circle" alt="" />
-        <div className="container">
-          {review.body}
-        </div>
-      </div>
-    ));
+  const reviews = this.props.reviews.map(review => (
```

Comment:
Hi Evelyn, 
I usually write the map function outside of the render to keep things clean. However, if it helps with testing, I can update it. I may need you to show me how you would rewrite it on Monday, because when I just tried to put the logic within the return itself, I get errors.
Thanks,
Priya